kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -631,16 +650,26 @@ struct GetValue {
     const auto &category = args.values[0];
     const auto key = concordUtils::hexToSliver(args.values[1]).toString();
     std::optional<categorization::Value> val;
+
+    std::unordered_map<std::string, std::string> outputMap;
     if (args.values.size() >= 3) {
       auto requested_block_version = toBlockId(args.values[2]);
       val = adapter.get(category, key, requested_block_version);
     } else {
       val = adapter.getLatest(category, key);
+      std::optional<categorization::TaggedVersion> taggedVersion = adapter.getLatestVersion(category, key);
+      if (taggedVersion) {
+        auto uint64Version = taggedVersion.value().version;
+        outputMap.insert(std::make_pair(std::string("version"), std::to_string(uint64Version)));
+      }
```

Comment:
I think we can make it uniform and return the version in both cases - when the user requests the latest value and when a version is specified.

Nit: Can use `const auto tagged_version = ` for simplicity.

Reply:
Done